src/cli_keystore/dev.js

```diff
@@ -17,5 +17,5 @@
  * under the License.
  */
 
-require('../setup_node_env');
+require('../setup_node_env/cli');
```

Comment:
Can we drop the new `cli` and instead require `no_transpilation`?

Reply:
Pushed with 7bcf044